installing/installing_ibm_z/installing-ibm-z-kvm.adoc

```diff
@@ -0,0 +1,95 @@
+[id="installing-ibm-z-kvm"]
+= Installing a cluster  with {op-system-base} KVM on IBM Z and LinuxONE
+include::modules/common-attributes.adoc[]
+:context: installing-ibm-z-kvm
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on
+IBM Z or LinuxONE infrastructure that you provision.
+
+[NOTE]
+====
+While this document refers only to IBM Z, all information in it also applies to LinuxONE.
+====
+
+[IMPORTANT]
+====
+Additional considerations exist for non-bare metal platforms. Review the information in the
+link:https://access.redhat.com/articles/4207611[guidelines for deploying {product-title} on non-tested platforms] before you install an {product-title} cluster.
+====
+
+== Prerequisites
+
+* Before you begin the installation process, you must clean the installation directory. This ensures that the required installation files are created and updated during the installation process.
+
+* Provision
+xref:../../storage/persistent_storage/persistent-storage-nfs.adoc#persistent-storage-nfs[persistent storage using NFS]
+for your cluster. To deploy a private image registry, your storage must provide
+ReadWriteMany access modes.
+* Review details about the
+xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update]
+processes.
+* If you use a firewall, you must
+xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configure it to allow the sites] that your cluster requires access to.
+* {op-system-base} KVM system that is hosted on the LPAR and based on {op-system-base} 8.3 or higher
+
+[NOTE]
```

Comment:
Can you nest this note in the bulleted list following the step with the link to the site list?